loadrunner_professional/assets/logs/loadrunner_professional.yaml

```diff
@@ -0,0 +1,61 @@
+id: loadrunner_professional
+metric_id: loadrunner_professional
+facets: null
```

Comment:
Can you confirm you don't want facets out of the box with your integration? For context, a log pipeline will help customers easily identify that the logs are coming from your integration, and will include out of the box facets to help customers understand what attributes are important to the integration and for them to quickly filter their logs.

Reply:
Hi @emarsha94, we confirm we do not want facets. All the related tags for the logs will be created via the pipeline.